reference/constraints/Json.rst

```diff
@@ -0,0 +1,100 @@
+Json
+=======
+
+Validates that a value is a ``json``. Specifically, this checks to see if
+the value is a valid ``json`` or not.
+
++----------------+-----------------------------------------------------------------------+
+| Applies to     | :ref:`property or method <validation-property-target>`                |
++----------------+-----------------------------------------------------------------------+
+| Options        | - `message`_                                                          |
+|                | - `payload`_                                                          |
++----------------+-----------------------------------------------------------------------+
+| Class          | :class:`Symfony\\Component\\Validator\\Constraints\\Json`             |
++----------------+-----------------------------------------------------------------------+
+| Validator      | :class:`Symfony\\Component\\Validator\\Constraints\\JsonValidator`    |
++----------------+-----------------------------------------------------------------------+
+
+Basic Usage
+-----------
+
+The ``Json`` constraint can be applied to a property or a "getter" method,
+but is most commonly useful in the latter case. For example, suppose that
+you want to guarantee that some ``jsonString`` property is a valid ``json``.
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Entity/Author.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Author
+        {
+            /**
+             * @Assert\Json(
+             *     message = "You've entered an invalid Json."
+             * )
+             */
+             protected $jsonString;
+        }
+
+    .. code-block:: yaml
+
+        # config/validator/validation.yaml
+        App\Entity\Book:
+            properties:
+                jsonString:
+                    - Json:
+                        message: You've entered an invalid Json.
+
+    .. code-block:: xml
+
+        <!-- config/validator/validation.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="App\Entity\Book">
```

Comment:
... and here `Book` again. Please make sure to use the same Entity for all examples.

And I prefer sth. like `$chapters` (if you use `Book`) or `$ratings` (if you use `Author`) instead of `$jsonString`

Reply:
Yes you are right, I've changed all Class to `Book`and `chapters` 👍